### ISSUE TITLE
while using --add_unary_ew_scales_for_dla leading to a great decrease in accuracy, if one input without int8 scales and larger than 128，set with scale 1.0 will overflow，how does quantization works in this situation?

### ISSUE THREAD

**Issue by Graham1025** [NVIDIA/TensorRT#3599]
## Description

while using --add_unary_ew_scales_for_dla leading to a great decrease in accuracy, if one input without int8 scales and larger than 128，set with scale 1.0 will overflow，how does quantization works in this situation?

## Environment

**TensorRT Version**: 8.6

**NVIDIA GPU**: orin

**NVIDIA Driver Version**:

**CUDA Version**: cuda-11.4

**CUDNN Version**:

Operating System:

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by zerollzeng** [comment]
Sorry I don't get what you mean, could you please elaborate which sample/demo you are referring? Would be great if you can provide a reproduce.

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, thanks all!
